src/api/comments.ts

```diff
@@ -0,0 +1,20 @@
+import { Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+
+export const getComments = (postId: Comment['postId']) => {
+  return client.get<Comment[]>(`
+/comments?postId=${postId}`);
```

Comment:
The template literal for the URL is split across two lines. Consider keeping it on a single line for better readability and to avoid potential issues with whitespace.